test/ProgressOnderwijsUtils.Tests/Data/ParameterizedSqlTest.cs

```diff
@@ -246,6 +246,18 @@ public void ParameterizedSqlDoesNotUseLiteralsEnumsMarked_IEnumShouldBeParameter
 
     [TestNotLiteral]
     enum ExampleNonLiteralEnum { SomeValue = 1, }
+
+    [Fact]
+    public void In_helper_works_for_empty_set()
+        => PAssert.That(() => SQL($"x.y").In(Array.Empty<int>()).CommandText() == "x.y in (select top(0) TVP.QueryTableValue from @par0 TVP)");
+
+    [Fact]
+    public void In_helper_works_for_single_item()
+        => PAssert.That(() => SQL($"x.y").In(new[] { 7, }).CommandText() == "x.y = @par0");
+
+    [Fact]
+    public void In_helper_works_for_multiple_items()
```

Comment:
We hebben ook een boel tests op een echte (maar tijdelijke) DB, dus dat zou hier ook niet gek zijn.